assets/css/obs.css

```diff
@@ -0,0 +1,9 @@
+.obs-edit-field-input-value {
+    display: inline-block;
+    margin-top: 10px;
+}
+
+.obs-edit-value {
+    display: inline-block;
+    margin-top: 8px;
```

Comment:
@jomadoye I'm going to go ahead and merge these in, but as a post-commit change can you rename "obs-edit-value" to "obs-view-value"? I think this makes more sense because this is the css that is used when the form is in "view" mode (ie, context.mode !== 'edit')

I'm not quite sure what the "ebs-edit-field-input-value" css and how it is different, but it looks like it is used in a "view" context as well?